src/Networks/Xds/Blockcore.Networks.Xds/XdsTest.cs

```diff
@@ -1,10 +1,206 @@
+                coinbaseMaturity: 50,
+                premineHeight: 0,
+                premineReward: Money.Coins(0),
+                proofOfWorkReward: Money.Coins(50),
+                targetTimespan: TimeSpan.FromSeconds(14 * 24 * 60 * 60),
+                targetSpacing: TimeSpan.FromSeconds(256),
+                powAllowMinDifficultyBlocks: false,
+                posNoRetargeting: false,
+                powNoRetargeting: false,
+                powLimit: new Target(new uint256("00000fffffffffffffffffffffffffffffffffffffffffffffffffffffffffff")),
+                minimumChainWork: null,
+                isProofOfStake: true,
+                lastPowBlock: 1_000_000_000,
+                proofOfStakeLimit: new BigInteger(uint256.Parse("00000fffffffffffffffffffffffffffffffffffffffffffffffffffffffffff").ToBytes(false)),
+                proofOfStakeLimitV2: new BigInteger(uint256.Parse("000000000000ffffffffffffffffffffffffffffffffffffffffffffffffffff").ToBytes(false)),
+                proofOfStakeReward: Money.Coins(50),
+                proofOfStakeTimestampMask: 0x0000003F // 64 sec
+            );
+
+            this.StandardScriptsRegistry = new XdsStandardScriptsRegistry();
+
+            this.Base58Prefixes = new byte[12][];
+            this.Base58Prefixes[(int)Base58Type.PUBKEY_ADDRESS] = new byte[] { 0 }; // deprecated - bech32/P2WPKH is used instead
+            this.Base58Prefixes[(int)Base58Type.SCRIPT_ADDRESS] = new byte[] { 5 }; // deprecated - bech32/P2WSH is used instead
+            this.Base58Prefixes[(int)Base58Type.SECRET_KEY] = new byte[] { 128 };
+            this.Base58Prefixes[(int)Base58Type.ENCRYPTED_SECRET_KEY_NO_EC] = new byte[] { 0x01, 0x42 };
+            this.Base58Prefixes[(int)Base58Type.ENCRYPTED_SECRET_KEY_EC] = new byte[] { 0x01, 0x43 };
+            this.Base58Prefixes[(int)Base58Type.EXT_PUBLIC_KEY] = new byte[] { 0x04, 0x88, 0xB2, 0x1E };
+            this.Base58Prefixes[(int)Base58Type.EXT_SECRET_KEY] = new byte[] { 0x04, 0x88, 0xAD, 0xE4 };
+            this.Base58Prefixes[(int)Base58Type.PASSPHRASE_CODE] = new byte[] { 0x2C, 0xE9, 0xB3, 0xE1, 0xFF, 0x39, 0xE2 };
+            this.Base58Prefixes[(int)Base58Type.CONFIRMATION_CODE] = new byte[] { 0x64, 0x3B, 0xF6, 0xA8, 0x9A };
+            this.Base58Prefixes[(int)Base58Type.ASSET_ID] = new byte[] { 23 };
+
+            var encoder = new Bech32Encoder(this.CoinTicker.ToLowerInvariant());
+            this.Bech32Encoders = new Bech32Encoder[2];
+            this.Bech32Encoders[(int)Bech32Type.WITNESS_PUBKEY_ADDRESS] = encoder;
+            this.Bech32Encoders[(int)Bech32Type.WITNESS_SCRIPT_ADDRESS] = encoder;
+
+            this.Checkpoints = new Dictionary<int, CheckpointInfo>();
+            this.DNSSeeds = new List<DNSSeedData>();
+            this.SeedNodes = new List<NetworkAddress>
+            {
+            };
+
+            RegisterRules(this.Consensus);
+        }
+
+        private static void RegisterRules(IConsensus consensus)
```

Comment:
same here call the rules from main (see btc network)

Reply:
you are right, I fixed that!